Dockerfile

```diff
@@ -6,7 +6,6 @@ LABEL vcs-url="https://github.com/grain-lang/grain"
 LABEL maintainer="philip@grain-lang.org"
 
 RUN apt-get update
-RUN apt-get install -y cmake=3.13.4-1
```

Comment:
Probably don't need the line above anymore either huh?

Reply:
Probably not. I wasn't sure if that updated like gcc or something. Do you think I should remove it?

Reply:
Yeah, let's remove it.

Reply:
Amended and force-pushed.